webapp/src/components/cardLimitNotification.tsx

```diff
@@ -79,7 +79,7 @@ const CardLimitNotification = () => {
         title = intl.formatMessage(
             {
                 id: 'notification-box-cards-hidden.title',
-                defaultMessage: 'This action hid another card',
+                defaultMessage: 'This action hide another card',
```

Comment:
'hid' is correct per https://github.com/mattermost/focalboard/pull/3151

```suggestion
                defaultMessage: 'This action hid another card',
```